mod_reforged/hooks/config/strings.nut

```diff
@@ -1176,7 +1183,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 				"Gain a stack whenever you end your [turn|Concept.Turn] without having used [Wait,|Concept.Wait] up to a maximum of 3 stacks.",
 				"Gain " + ::MSU.Text.colorPositive("+15") + " [Resolve|Concept.Bravery] and [Initiative|Concept.Initiative] while you have at least 1 stack.",
 				"Skills build up " + ::MSU.Text.colorPositive("15%") + " less [Fatigue|Concept.Fatigue] while you have at least 2 stacks.",
-				"Deal " + ::MSU.Text.colorPositive("15%") + " more damage while you have 3 stacks.",
+				"Deal " + ::MSU.Text.colorPositive("15%") + " additional damage while you have 3 stacks.",
```

Comment:
`more` is correct here. We should reserve the term `additional` for things which additively change a modifier.